src/pre_commit_vauxoo/pre_commit_vauxoo.py

```diff
@@ -45,6 +47,24 @@ def get_files(path):
     return ls_files
 
 
+def get_uninstallable_modules(src_path) -> set:
+    """Find all odoo modules that are set as not installable. They must have a key 'installable' with a False value
+    in order to be considered not installable.
+
+    :return: A set of strings, each one representing the relative path (from repo dir) to an uninstallable module.
+    """
+    results = set()
+    for path in glob.glob(os.path.join(src_path, "**/__manifest__.py"), recursive=True):
+        with open(path) as manifest:
+            try:
+                if not ast.literal_eval(manifest.read()).get("installable", True):
+                    results.add(os.path.dirname(os.path.relpath(path, start=src_path)))
+            except (ValueError, TypeError, SyntaxError):
```

Comment:
`[].get` raises `AttributeError`

```suggestion
            except (ValueError, TypeError, SyntaxError, AttributeError):
```